### PR TITLE
last fixes

### PR DESCRIPTION
**Pull request by tbazant**
### Description

Last fixes coming from smartdocs review

### CODE REVIEW

File: articles/network-interface-predictable-naming.asm.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
       <abstract>
         <variablelist>
           <varlistentry>
-            <term>TOPIC</term>
+            <term>WHAT</term>
```
Comment (janajaeger) [inline articles/network-interface-predictable-naming.asm.xml:87]:
```suggestion
            <term>WHAT?</term>
```

File: articles/network-interface-predictable-naming.asm.xml [hunk 2]
```diff
@@ -94,7 +94,7 @@
             </listitem>
           </varlistentry>
           <varlistentry>
-            <term>INTENTION</term>
+            <term>WHY</term>
```
Comment (janajaeger) [inline articles/network-interface-predictable-naming.asm.xml:97]:
```suggestion
            <term>WHY?</term>
```

File: glues/network-interface-predictable-naming-troubleshooting.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ xmlns:its="http://www.w3.org/2005/11/its"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
+    <title>Troubleshooting</title>
+    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
+  </info>
+  <para>
+    The following sections describe common problems to network interface naming
```
Comment (janajaeger) [inline glues/network-interface-predictable-naming-troubleshooting.xml:19]: Following sections? There's just one :(  According to the styleguide, we don't tolerate lone sections. Could you either think of another possible issue and cover it or massage the lone section into the parent section?

File: glues/network-interface-predictable-naming-troubleshooting.xml
Comment (janajaeger) [inline glues/network-interface-predictable-naming-troubleshooting.xml:1]: Why is this a glue? Not sure we made any troubleshooting-related rules yet, but I'd see troubleshooting either as a task or a reference topic. Frank has gone for task, the sudo crowd opted for reference. I guess we should discuss this and fix it once an for all :)

**Review by janajaeger — COMMENTED** [review]
Just some minor nitpicks.

File: glues/network-interface-predictable-naming-troubleshooting.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <para>
+          The information provided to udev is not suitable, such as non-unique
+          device IDs.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The solution is to create custom naming udev rules for the devices that
+      are not properly named by the udev rules that are currently in use.
+      Inspect file in the <filename>/etc/udev/rules.d</filename> directory, for
```
Comment (dariavladykina) [inline glues/network-interface-predictable-naming-troubleshooting.xml:46]:
```suggestion
      Inspect files in the <filename>/etc/udev/rules.d</filename> directory, for
```

**Review by dariavladykina — COMMENTED** [review]
Just a small nitpick here. Thanks!

File: glues/network-interface-predictable-naming-troubleshooting.xml
Comment (tbazant) [reply glues/network-interface-predictable-naming-troubleshooting.xml:1]: i'll go for reference then, makes more sense to me, ok?

**Pull request merged**